example-usage/README.md

```diff
@@ -0,0 +1,92 @@
+# Dev Container CLI Examples
+
+This repository provides a basic examples of using the devcontainer CLI for different use cases. It includes example scripts to:
+
+1. [Use three different tools from a development container](#tool-examples)
+2. [Use a dev container as your CI build environment to build an application](#ci-build-environment-example) (even if it is not deployed as a container)
+3. [Build a container image rom a devcontainer.json file](#building-an-image-from-devcontainerjson) that includes [dev container features](https://containers.dev/implementors/features/)
+
+Each should run on macOS or Linux (including WSL2 on Windows).
+
+## Pre-requisites
+
+1. Install Node.js 16 (e.g., using [nvm](https://github.com/nvm-sh/nvm))
+2. Install [node-gyp pre-requisites](https://github.com/nodejs/node-gyp):
+   - **Linux:** Use your distro's package manager. E.g. on Ubuntu/Debian: `sudo apt-get update && sudo apt-get install python3-minimal gcc g++ make`
+   - **macOS:** Install the XCode Command Line Tools ([more info](https://github.com/nodejs/node-gyp/blob/main/README.md#on-macos))
+3. Make sure you have an OpenSSH compliant `ssh` command available and in your path if you plan to use the `Vim via SSH` example. (It should already be there on macOS, and in Linux/WSL, you can install `openssh-client` using your distro's package manager if its missing.)
+3. Install the latest dev container CLI: `npm install -g @devcontainers/cli`
+
+## Using the examples
+
+All examples use the contents of the `workspace` folder for their configuration if you want to make any modifications. The example scripts are then in different sub-folders. 
+
+### Tool examples
+
+You can use the example by opening a terminal and typing one of the following:
+
+-  `tool-vscode-server/start.sh` - [VS Code Server](https://code.visualstudio.com/docs/remote/vscode-server) (official)
+- `tool-openvscode-server/start.sh` - [openvscode-server](https://github.com/gitpod-io/openvscode-server)
+- `tool-vim-via-ssh/start.sh` - Vim via a SSH connection. SSH is used primarily to demonstrate how this could be achieved from other SSH supporting client tools.
```

Comment:
```suggestion
- `tool-vim-via-ssh/start.sh` - Vim via an SSH connection. SSH is used primarily to demonstrate how this could be achieved from other SSH supporting client tools.
```